.gitignore

```diff
@@ -23,4 +23,5 @@ test-run-details.json
 cypress/fixtures/test-run-details.json
 env_variables/*.env.json
 env_variables/*
-!env_variables/template.env.json
\ No newline at end of file
+!env_variables/devClusterTemplate.env.json
```

Comment:
can we have a uniform naming for both files as staging is also on some cluster? something like `devEnvTemplate` and `stagingEnvTemplate`

Reply:
File name already has .env.json, so no need to add Env in the file name again, whereas only devTemplate / stagingTemplate would be suffice.